common/src/main/resources/assets/voicechat/lang/pt_br.json

```diff
@@ -1,25 +1,25 @@
 {
   "key.categories.voicechat": "Voice Chat",
-  "key.push_to_talk": "Mantenha pressionado para falar",
+  "key.push_to_talk": "Mantenha pressionado p/ falar",
```

Comment:
Why did you change this?

Reply:
In Portuguese-Brazilian "Para" can be abbreviated to "p/", this simplifies and does not lose meaning.

Reply:
This helps to make the text smaller, which makes the buttons prettier and doesn't lose meaning in translation.